src/main/kotlin/net/atos/zac/zoeken/IndexingService.kt

```diff
@@ -49,8 +49,11 @@ class IndexingService @Inject constructor(
         private const val SOLR_MAX_RESULTS = 100
         private const val TAKEN_MAX_RESULTS = 50
 
+        const val MAX_SEQUENTIAL_ERRORS = 20
```

Comment:
```suggestion
        private const val MAX_SEQUENTIAL_ERRORS = 20
```

Better not to reference this directly from test code I think. See below.